### DISCUSSION TITLE
Option to make discussions "sticky" per category.

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#11059] [Discussions]
Right now can you pin discussions, which makes them appear at the top.

While pinning is a nice option, does it have clear limits, especially with discussions that have a lot of categories, such as this one here.

Because of this would I like to suggest to have an option to make discussions "sticky".

A "sticky" discussion wouldn't show up at the top like a pinned one, but would "stick" to the top of its respective category.

This would allow maintainers to keep important discussions such as a welcome or "Read me" discussion at the top of a category for easy finding while keeping the slots for pinned discussions free for recent news.

tl;dr: An option to make discussions sticky for specific categories, similar to how you have sticky threads on the GitHub Community forum or other forums (i.e. XenForo-based forums).
